src/main/resources/jenkins/plugins/extracolumns/DisableProjectColumn/column.jelly

```diff
@@ -53,4 +51,6 @@ THE SOFTWARE.
             <st:adjunct includes="jenkins.plugins.extracolumns.DisableProjectColumn.button"/>
        </j:if>
     </td> 
+
+    <st:adjunct includes="jenkins.plugins.extracolumns.DisableProjectColumn.disable-project"/>
```

Comment:
The following warning appears in the logs:
```
WARNING o.k.s.f.adjunct.AdjunctsInPage#findNeeded: No such adjunct found: 
jenkins.plugins.extracolumns.DisableProjectColumn.disable-project
org.kohsuke.stapler.framework.adjunct.NoSuchAdjunctException:
Neither jenkins.plugins.extracolumns.DisableProjectColumn.disable-project.css, .js, .html, nor .jelly were found
```
Not sure what the intention was, but I guess it works just fine without this adjunct.

Reply:
Thanks for pointing it, I completely overlooked it.
It was leftover of other changes and I removed that line.

In first version I also wanted to included change about CSP, but I was beaten to it by https://github.com/jenkinsci/extra-columns-plugin/pull/33

Reply:
No worries. Thanks again for the fix and your PR.